### DISCUSSION TITLE
need help? post a new question!

### DISCUSSION THREAD

**Discussion by hakusaro** [Pryaxis/TShock#1695] [General]
The new rule of thumb is this: if you need a question answered about TShock, you should ask your question here. If need help setting up TShock, find a bug, or encounter any other problem, the best place to ask for help is here.

Why here?

1. If your question gets answered, it can be marked as answered, like on Stack Overflow. We'll help you out and get you on the right path. If someone else asks a similar question, we can link them to your question (ideally, they don't even need to post).

2. If you ask a question on Discord or Telegram, it'll likely be lost, and it won't help other people. It's hard to find items in the history and we can't expect every person to search a chat log for help.

3. GitHub isn't going away any time soon :)

**Comment by cccdfern** [comment]
Especially number 2, hopefully this will mean questions do not need to be asked over and over, in theory.

**Comment by tltl6324** [comment]
thank you for your developing. 
I have a question. 

i want to know how to protect my map.
for example. in case that if anyone touch or break a block. the doing is denied

**Reply by hakusaro** [reply]
Please post again by creating a new discussion topic, sorry!
